app/assets/stylesheets/layout/_main_menu.sass

```diff
@@ -194,30 +196,36 @@ $toggler-width: 40px
     float: left
 
   #toggle-project-menu
-    border-width: $main-menu-item-border-width 0 $main-menu-item-border-width $main-menu-item-border-width
-    border-style: solid
-    border-color: $main-menu-item-border-color
-    height: $main-menu-item-height
-    width: calc(#{$main-menu-width} - 1px)
+    border: none
+    background: none
+    height: 100vh
+    width: 18px
+    position: fixed
+    top: 0px
+    left: calc(#{$main-menu-width} - 1px)
```

Comment:
Why is it necessary to subtract one pixel?

Reply:
This on pixel is to overlap make the toggles left border (the highlight) overlap the right border of the menu.

Reply:
Ah I see. Good one 👍